pl-PL.json

```diff
@@ -1,6 +1,15 @@
 {
-  "0.3% LP Fee": "0.3% LP Opłaty",
   "1 {0} = {1} {2}": "1 {0} = {1} {2}",
+  "A collateral is required in order to interact with smart contracts on Cardano. The collateral amount should be greater than": "A collateral is required in order to interact with smart contracts on Cardano. The collateral amount should be greater than",
+  "Collateral not found!": "Collateral not found!",
+  "Fee paid for the service of off-chain Laminar batcher to process transactions.": "Fee paid for the service of off-chain Laminar batcher to process transactions.",
+  "For each trade a 0.3% goes to liquidity providers.": "For each trade a 0.3% goes to liquidity providers.",
+  "Status": "Status",
+  "TVL": "TVL",
+  "The amount is separated from your account balance, you can choose to return it to your balance at any time.": "The amount is separated from your account balance, you can choose to return it to your balance at any time.",
+  "Zap": "Zap",
+  "Zap In": "Zap In",
+  "0.3% LP Fee": "0.3% LP Opłaty",
```

Comment:
@baymac  I think we should maintain sort order for keys, this will reduce potential conflicts when both side are sorting.

Reply:
what do u mean? currently the lines are partitioned into English (above) and translated (below), and for each partition keys are sorted

Reply:
ok get it, this sorted alphabetically with prioritizing untranslated ones.

There is very very minor issue with above/below handcrafting since some translation are 1:1 and there is no additional bit of information that value for key was really translated ;) i. e. "Status" in polish is "Status" so it was translated from beginning and should be below then. And some like "TVL" should stay probably same everywhere, since this is practically term to learn.

I think I would just prefer sorted by key and key only, since I can compare easily with other translations using meld for get additional keys / compare with original english template line by line.